client/js/verification-portal/pregnancy-resource-center/get-resource-to-verify-button.js

```diff
@@ -87,7 +87,8 @@ const populateForm = ({ changeFieldValue, resource }) => {
 		_.get(verifiedData, 'primaryContactPerson.verified'),
 	)
 
-	_.forEach(services, service => {
+	console.log('SERVICES', pregnancyCenterServices)
```

Comment:
I think we want to remove the log here, right?

Reply:
Eek good catch!